galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -274,8 +275,8 @@ def get(self, request, *args, **kwargs):
             )
 
         metrics.collection_artifact_download_failures.labels(status=response.status_code).inc()
-        raise APIException('Unexpected response from content app. '
-                           f'Code: {response.status_code}.')
+        raise APIException(_('Unexpected response from content app. '
+                             'Code: %s.' % response.status_code))
```

Comment:
It seems most places the string variable is outside the `gettext_lazy` method, but at least 2 places in this file have it inside... should it always be outside?
inside: `gettext_lazy('Unexpected response from content app. Code: %s.' % response.status_code)`
outside: `gettext_lazy('Collection %s not found') % version_str`

@jerabekjiri do you regenerate the `locale/*/LC_MESSAGES/django.po` files after each code edit? That might be a good test to see if it is supported. For example the .po item related to this is: `msgid "Unexpected response from content app. "`

Line 278 & 279 could probably fit on one line and be easier to read / edit in future.

Reply:
> It seems most places the string variable is outside the gettext_lazy method, but at least 2 places in this file have it inside... should it always be outside?

Seems both versions work and generate the right strings, but I'll change it to outside, according to Django documentation.

I'll regenerate the messages and push changed files, I forgot to renew them :)

Reply:
thanks, looks good